src/subscript/sunsch/sunsch.py

```diff
@@ -551,6 +552,48 @@ def substitute(insert_statement):
     return resultfilename
 
 
+def wrap_long_lines(string, maxchars=128, warn=True):
+    """Wrap long lines in a multiline string.
+
+    Short enough lines are not touched.
+
+    Args:
+        string (str): Multiline string to be possibly wrapped
+        maxchars (int): Maximal length of each line
+        warn (bool): Whether to log a warning for each line
+            of excessive length
+
+    Returns:
+        str: Multiline string with no lines more than maxchars
+        in length. Trailing whitespace is always stripped.
+    """
+    wrappedstr = ""
+    for line_idx, line in enumerate(string.split("\n")):
+        commentsplit = line.split("--")
```

Comment:
Some may have more than one set of `--` (if they are eager on commenting in and out), suggest to use partition instead of split, or add the comment again further down as `"--".join(commentsplit[1:]"`

Reply:
Or `line.split("--", 1)` would also do the trick

Reply:
`partition` made it prettier.